### ISSUE TITLE
Priority of flags vs fly.toml not consistent for image vs dockerfile

### ISSUE THREAD

**Issue by catflydotio** [superfly/flyctl#1952]
v0.0.499

Specify an image in `[build]` in fly.toml AND use the `--image` flag: the flag takes precedence

Specify a dockerfile in `[build]` in fly.toml AND use the `--dockerfile` flag: the value from fly.toml takes precedence

(related but not the same: https://github.com/superfly/flyctl/issues/1950 where an image specified anywhere overrides a dockerfile specified anywhere)

**Comment by catflydotio** [comment]
To be clear, I would think that supplying a flag indicates the intention to override the content of fly.toml.

**Comment by tvdfly** [comment]
I agree that providing a flag should take precedent over a value in fly.toml.
